lib/src/constants/constant_colors.dart

```diff
@@ -1,5 +1,7 @@
 import 'package:flutter/material.dart';
 
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+
 class AppColor {
   static const green = Color(0xff30BE76);
```

Comment:
What if the customer change from green to pink? I think you should use mainTheme or appTheme instead of green.

Reply:
I will talk to the group about this issue